src/components/badge/Badge.svelte

```diff
@@ -26,7 +26,7 @@
     <slot />
   </a>
 {:else}
-  <span class={cssClass} on:click {style} data-type={type}>
+  <span class={cssClass} on:click on:keydown role="button" tabindex={0} {style} data-type={type}>
```

Comment:
giver det mening at det er en span?
det er vel kun en span for ikke at skulle restyle en button? og med button skal man ikke tænke på at sætte begge interaktive funktioner
det at de begge udstilles gør dem jo ikke obligatoriske, så keydown bliver bare glemt i parent i stedet for her

Reply:
Hmm ja, en hel anden ting, hvorfor har vi interactive funktionalitet på et badge? Det ser ikke ud til vi bruger on:click på et Badge nogle steder? Så spørgsmålet er om man kan fjerne alt den interactive funktionalitet

Reply:
se evt kommentar med løsningsforslag længere nede ved card

Reply:
Et badge er vel som udgangspunkt uden funktionalitet og nærmere en styling?

Reply:
@mbjedk så hvis jeg forstår dig rigtigt bør der egentlig slet ikke være mulighed for interaktion med et badge, men man skal lave det som en wrapper?

Reply:
Eller bare have en badge class og slette Svelte-componentet. Det kan jo ikke noget.

Reply:
Synes det giver mening, at fjerne interactive funktionalitet fra Badge, men synes godt vi kan lade componentet være, det giver en fin letlæselig kode i Svelte.

Reply:
Synes det giver bedre mening, at benytte standard-HTML-elementer, så man faktisk kan se hvad badge´t gør og samtidig ikke skal udstille alt muligt.

Hvad hvis jeg gerne vil have, at linket på et badge åbner i et nyt vindue? No can do, uden at rette i designsystemet.

`<a href="#" class="badge" target="_blank">Linktekst</a>`
Det er også rimelig letlæseligt.

Det betyder samtidig, at man kan benytte svelte-actions på badges.